packages/kbn-optimizer/limits.yml

```diff
@@ -95,7 +95,7 @@ pageLoadAssetSize:
   management: 46112
   maps: 90000
   mapsEms: 26072
-  metricsDataAccess: 60000
+  metricsDataAccess: 91984
```

Comment:
Did we look into keeping these changes async?

Reply:
I haven't considered it tbh will check